javascript/dist/i18n.js

```diff
@@ -132,7 +132,7 @@
 						return match;
 					}
 
-					return subMatch1 + params[i += 1];
+					return subMatch1 + params[i++];
```

Comment:
I think I forgot to ask someone to build dist files in a prior PR... oh well this fixes it!